iaso/utils/__init__.py

```diff
@@ -127,3 +127,7 @@ def is_mobile_request(request):
     user_agent = request.META.get("HTTP_USER_AGENT", "")
     # Check if the User-Agent string contains keywords typical of mobile devices
     return any(mobile in user_agent.lower() for mobile in ["mobile", "android", "iphone", "ipad", "windows phone"])
+
+
+def is_multi_account_user(user):
+    return hasattr(user, "tenant_user")
```

Comment:
I guess we can't alter the `User` class directly because it belongs to the `django.contrib.auth` application…

[`add_to_class()`](https://code.djangoproject.com/wiki/DevModelCreation#InstallingTheAttributes) could be an option here but it's an internal function and it's a bit of a monkey patch.

In the end, an util function does the job very well 👍